chaste_codegen/model_with_conversions.py

```diff
@@ -11,17 +11,13 @@
     Expr,
     Float,
     Function,
-    Pow,
-    Wild,
-    log,
     simplify,
     sympify,
 )
-from sympy.codegen.cfunctions import log10
-from sympy.codegen.rewriting import ReplaceOptim, log1p_opt, optimize
 
 from chaste_codegen import LOGGER, CodegenError
 from chaste_codegen._math_functions import MATH_FUNC_SYMPY_MAPPING
+from chaste_codegen._optimize import optimize_expr
```

Comment:
It might be good to rethink the organisation a bit?

Usually the `_hidden_modules` are only used when their methods are imported by some _library_, which then exposes them as part of the public API. Chaste codegen seems to be an application rather than a library, so maybe this isn't the best way to do it?